apps/web/src/components/Basenames/RegistrationForm/index.tsx

```diff
@@ -32,7 +28,7 @@ import { formatEther, zeroAddress } from 'viem';
 import { useAccount, useBalance, useReadContract, useSwitchChain } from 'wagmi';
 import { RegistrationButton } from './RegistrationButton';
 
-function formatEtherPrice(price?: bigint) {
+export function formatEtherPrice(price?: bigint) {
```

Comment:
let's put these formatting fns into `apps/web/src/utils` or `apps/web/src/utils/basenames`. i'd probably do the former, but okay with the latter since they're currently only used by basenames-related functionality

Reply:
Done [020fef9](https://github.com/base/web/pull/2410/commits/020fef973e40c535d568184675f704ec4681a38f)